components/presenters/Docs/Sidebar/Sidebar.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import { Sidebar, SidebarMenu, SidebarMenuItem } from '.'
+
+const stories = storiesOf('Docs/Sidebar', module)
+
+stories.add('Default', () => (
+  <Sidebar title="Components">
+    <SidebarMenu>
+      <SidebarMenuItem href="#alert" title="Alert" />
```

Comment:
I wonder if instead of href prop this works nicely like this:

```jsx
import Link from 'next/link'

<SidebarMenuItem link={<Link href="#example-title">Example Title</Link>} />
```

https://nextjs.org/docs/api-reference/next/link

Reply:
Thank you 🙏  .. this was starting to bother me!